### PR TITLE
Host checks execution frontend

### PR DESCRIPTION
**Pull request by EMaksy**
# Description
This PR will enable the host checks execution in the UI

## Demo
demo_checks_execution

## How was this  tested?
Added automated tests

### CODE REVIEW

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a tiny thing then we can merge I guess

File: assets/js/components/HostDetails/HostChecksSelection.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ function HostChecksSelection({
   onSaveSelection,
   onSelectedChecksChange,
   hostChecksExecutionEnabled,
+  startExecution = () => {},
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply assets/js/components/HostDetails/HostChecksSelection.jsx:25]: Sure!

File: assets/js/components/HostDetails/HostChecksSelection.jsx [hunk 2]
```diff
@@ -46,7 +47,9 @@ function HostChecksSelection({
             <Button
               type="primary"
               className="mx-1"
-              onClick={() => {}}
+              onClick={() => {
+                startExecution();
+              }}
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply assets/js/components/HostDetails/HostChecksSelection.jsx]: Actually this looks much better `onClick={onStartExecution}`  thanks!

**Review by nelsonkopliku — COMMENTED** [review]
Before merging this we need to get both wanda and the api merged.

**Review by dottorblaster — APPROVED** [review]
LGTM

File: assets/js/components/HostDetails/HostSettingsPage.test.jsx [hunk 3]
```diff
@@ -70,4 +70,44 @@ describe('HostSettingsPage component', () => {
...
+    const [StatefulHostSettingsPage] = withState(<HostSettingsPage />, state);
+
+    renderWithRouterMatch(StatefulHostSettingsPage, {
+      path: 'hosts/:hostID/settings',
+      route: `/hosts/${hostID}/settings`,
+    });
+    const startExecutionButton = screen.getByText('Start Execution');
+    expect(startExecutionButton).toBeDisabled();
+  });
+  it('should render HostSettingsPage with an enabled start execution button, as checks are selected', () => {
```
Comment (nelsonkopliku) [inline assets/js/components/HostDetails/HostSettingsPage.test.jsx:94]: just a newline

**Review by nelsonkopliku — APPROVED** [review]

File: assets/js/components/HostDetails/HostSettingsPage.test.jsx (hunk 3 above)
Comment (EMaksy) [reply assets/js/components/HostDetails/HostSettingsPage.test.jsx:94]: done :D

File: assets/js/state/actions/lastExecutions.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 export const UPDATE_LAST_EXECUTION = 'UPDATE_LAST_EXECUTION';
 export const EXECUTION_REQUESTED = 'EXECUTION_REQUESTED';
+export const HOST_EXECUTION_REQUESTED = 'HOST_EXECUTION_REQUESTED';
```
[the remark this answers is not in the document]
Comment (EMaksy) [reply assets/js/state/actions/lastExecutions.js:3]: Yeah this is better thanks!

**Pull request merged**